src/session/content/send_photo_dialog.rs

```diff
@@ -134,6 +134,8 @@ impl SendPhotoDialog {
         let paintable = imp.picture.paintable().unwrap();
         let width = paintable.intrinsic_width();
         let height = paintable.intrinsic_height();
+        imp.caption_entry.set_chat(Some(chat.to_owned()));
```

Comment:
I know it's a hacky way it does work unlike binding chat in template file :/
![screenshot](https://user-images.githubusercontent.com/58047802/187710523-bb54d8d6-48ea-45d3-8e44-ae686f57adbe.png)